.k8s/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG PYTHON_VERSION
+FROM docker.elastic.co/beats-dev/apm-agent-python:${PYTHON_VERSION}
```

Comment:
This is the proposal to cache the existing docker images defined in `./tests/Dockerfile` to speed up the builds